src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparatorPreviousLineEmptiness.java

```diff
@@ -31,4 +31,13 @@ public static int increment(int value) {
 
         return value * MULTIPLICATOR + 1;
     }
+
+    void foo2() {
+    }
+
+
+    void foo3() {
+
+
+    }
```

Comment:
these 2 methods above cause violation or not? please add violation or ok comments

Reply:
Done

Reply:
is there a vioaltion for foo2 method?

Reply:
No sir. There would be a violation when we use config "allowMultipleEmptyLines" = "false"`

Reply:
Added config to file, and put `// ok` comment to it.